ullyses/coadd.py

```diff
@@ -262,13 +262,16 @@ def get_coords(self):
         decs = list(set([h["dec_targ"] for h in self.primary_headers]))
         avg_ra = np.average(ras)
         avg_dec = np.average(decs)
-        return avg_ra, avg_dec
+        epoch = self.combine_keys("equinox")
+        if epoch == "MULTI":
+            epoch = "UNKNOWN"
+        return avg_ra, avg_dec, epoch
```

Comment:
Epoch and equinox refer to 2 different things: equinox (usually 2000.0) refers to the date at which the precessed coordinates are calculated at.  The precession of all neighboring objects (including guide stars) will basically be the same.  In contrast, epoch refers to the date at which the coordinates of the star were measured.  Proper motion will move the target relative to the local coordinate system.  So the actual coordinates of the target will change depending on when the object is observed (this should be the ra_aper and dec_aper coords).  I don't know what the standard way of dealing with this is - perhaps get some guidance from the team?

Reply:
I was going off the info here on how to name the keyword but I will check with other folks!
https://heasarc.gsfc.nasa.gov/docs/fcg/standard_dict.html
In particular, this definition seems to equate epoch and equinox.
```
KEYWORD:   EPOCH
REFERENCE: [FITS Standard](http://archive.stsci.edu/fits/fits_standard/) 
STATUS:    reserved
HDU:       any
VALUE:     real
COMMENT:   equinox of celestial coordinate system
DEFINITION: The value field shall contain a floating point number
giving the equinox in years for the celestial coordinate system in
which positions are expressed.  Starting with Version 1, the Standard
has deprecated the use of the EPOCH keyword and thus it shall not be
used in FITS files created after the adoption of the standard; rather,
the EQUINOX keyword shall be used.
```

Reply:
Also, the "true RA/Dec" of our targets is problematic in general, because we are combining data that can span decades in some cases. In the end, the coordinates we choose to put in the header are not always going to be the ones specified for each constituent observation.

Reply:
okay, got clarify from the team and I will rename the keyword to `EPOCH` as this is actually the correct term to describe the coords that are put in the header. Alex says " Epoch is more correct, since it denotes the time (mean time) at which the Gaia DR3 coordinates were determined.   They have not been precessed to the standard J2000 equinox. " In this case, it is better to ignore FITS standard. I'll update the code!